README.md

```diff
@@ -2,42 +2,45 @@
   <img src="images/logo.png" alt="Rooibos test framework" width="200" height="200"/>
 </p>
 <h3 align="center">
-Simple, mocha-and junit inspired, flexible, fun Brightscript test framework for ROKU apps
+Simple, mocha and junit inspired, flexible, and fun Brightscript test framework for Roku apps
 </h3>
 <p align="center">
   <img src="images/exampleImage.png" alt="Mocha test framework" />
 </p>
 
-[![build](https://img.shields.io/github/workflow/status/georgejecook/rooibos/build.svg?logo=github)](https://github.com/georgejecook/rooibos/actions?query=workflow%3Abuild)
+[![build](https://img.shields.io/github/actions/workflow/status/georgejecook/rooibos/build.yml?logo=github&branch=master)](https://github.com/georgejecook/rooibos/actions/workflows/build.yml)
 [![GitHub](https://img.shields.io/github/release/georgejecook/rooibos.svg?style=flat-square)](https://github.com/georgejecook/rooibos/releases)
 [![NPM Version](https://badge.fury.io/js/rooibos-roku.svg?style=flat)](https://npmjs.org/package/rooibos-roku)
 [![Coverage Status](https://coveralls.io/repos/github/georgejecook/rooibos/badge.svg?branch=master)](https://coveralls.io/github/georgejecook/rooibos?branch=master)
 
 ## Links
- - **[Documentation](https://github.com/georgejecook/rooibos/blob/master/docs/index.md)**
- - **[API Documentation](https://georgejecook.github.io/rooibos)**
- - **[CHANGELOG](CHANGELOG.md)**
- - **[VSCode snippets](docs/vsCodeSnippets.md)**
- - [Roku Developers slack](https://join.slack.com/t/rokudevelopers/shared_invite/zt-4vw7rg6v-NH46oY7hTktpRIBM_zGvwA)
+
+- [Documentation](https://github.com/georgejecook/rooibos/blob/master/docs/index.md)
+- [API Documentation](https://georgejecook.github.io/rooibos)
+- [CHANGELOG](CHANGELOG.md)
+- [VSCode snippets](docs/vsCodeSnippets.md)
+- [Roku Developers slack](https://join.slack.com/t/rokudevelopers/shared_invite/zt-4vw7rg6v-NH46oY7hTktpRIBM_zGvwA)
 
 ## Development
 
-Rooibos is an independent open-source project, maintained exclusively by volunteers.
+Rooibos is an independent open-source project maintained exclusively by volunteers.
 
-You might want to help! Get in touch via the slack group, or raise issues.
+You might want to help! Get in touch via the [slack group](https://join.slack.com/t/rokudevelopers/shared_invite/zt-4vw7rg6v-NH46oY7hTktpRIBM_zGvwA) or [raise issues](https://github.com/georgejecook/rooibos/issues/new).
 
 ## Sample project
-https://github.com/georgejecook/rooibos-roku-sample
+
+<https://github.com/georgejecook/rooibos-roku-sample>
```

Comment:
What's the point of these `<` `>` chars? I don't think it's of much benefit.

Reply:
Because of this linting rule - [MD034](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md034.md)

TL;DR - `Rationale: Without angle brackets, the URL isn't converted into a link by many Markdown parsers.`

I can change it if you'd like.

Reply:
Yeah, I think this one isn't quite as intuitive, and most of us are editing and consuming these exclusively within a github/vscode context so the `<` and `>` are just kind of in the way. :) So let's remove them for this PR.